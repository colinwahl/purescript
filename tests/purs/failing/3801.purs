-- @shouldFailWith ErrQualifierLexeme
module Main where

import Prelude as Prelude

x :: Int
x = Prelude.
