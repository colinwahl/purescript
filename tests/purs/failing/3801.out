Error found:
at tests/purs/failing/3801.purs:6:13 - 6:14 (line 6, column 13 - line 6, column 14)

  Unable to parse module:
  Unexpected whitespace character U+000A, expected qualifier


See https://github.com/purescript/documentation/blob/master/errors/ErrorParsingModule.md for more information,
or to contribute content related to this error.

